susemanager/src/mgr_bootstrap_data.py

```diff
@@ -353,6 +353,34 @@
     "libunwind",
 ]
 
+PKGLISTUBUNTU = [
+    "python3-tornado",
+    "libnorm-dev",
+    "pypy-zmq",
+    "libnorm1",
+    "python3-zmq-dbg",
+    "python-tornado",
+    "python-zmq",
+    "libpgm-5.2-0",
+    "python-zmq-dbg",
+    "libzmq3-dev",
+    "libpgm-dev",
+    "python3-systemd",
+    "python3-zmq",
+    "libzmq5",
+    "python-systemd",
+    "salt-master",
```

Comment:
We only ship salt-common and salt-minion. Master, api, ssh, syndic or cloud are not needed.